packages/css/src/components/file-input/file-input.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset-button {
+  border: 0;
+}
+
+.ams-file-input {
```

Comment:
I think we should add a border around the entire clickable area, to make it more obvious that it's clickable

Reply:
It was looking too much like a text input before?

Reply:
This is not a very clear comment, but I think you mean: adding a border to it will make it look too much like a text input?

If so: not necessarily. The preliminary Figma design uses a dashed border to indicate the clickable area. We could use that here as well.

Reply:
The question mark is because last week you asked me to remove the border.

Reply:
I've added a dasheb border and padding, I had to remove the `width: 100%` because the width of the button was being added to input. Turns out the buttons is like a shadow-dom iframe-ish thing: https://css-tricks.com/almanac/selectors/f/file-selector-button/#aa-the-button-is-part-of-the-shadow-dom